libraries-5/pom.xml

```diff
@@ -102,21 +109,25 @@
             <groupId>com.linecorp.armeria</groupId>
             <artifactId>armeria-graphql</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>nl.basjes.parse.useragent</groupId>
+            <artifactId>yauaa</artifactId>
+            <version>${yauaa.version}</version>
+        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
-            <version>${springBoot.version}</version>
+            <version>${spring-boot.version}</version>
             <scope>test</scope>
         </dependency>
     </dependencies>
 
     <properties>
         <jetcache.version>2.7.6</jetcache.version>
-        <springBoot.version>3.3.0</springBoot.version>
-        <spring.version>6.1.8</spring.version>
+        <spring-boot.version>3.1.0</spring-boot.version>
```

Comment:
Any reason that we went from 3.3.0 to 3.1.0 here? Prefer latest usually - same for other versions

Reply:
Fixed.

Reply:
I was getting a lot of incompatible version errors due to the jetcache dependencies, spent some time trying to fix it but eventually decided to downgrade the version. 

I suspect this was the same reason the spring context was downgraded by the original committer.

Reply:
Fixed.